packages/gateway/test/metrics.spec.js

```diff
@@ -70,6 +77,26 @@ test('Gets Metrics content', async (t) => {
   const response = await mf.dispatchFetch('http://localhost:8787/metrics')
   const metricsResponse = await response.text()
 
+  // content length of responses is 23
```

Comment:
? is zero below

Reply:
What do you mean below? Want to get the metrics before the requests and see if content length is zero?

Reply:
added that case